.github/workflows/ci-mount-inspect.yaml

```diff
@@ -0,0 +1,83 @@
+# Run weekly on main, Sundays at 23:00 UTC (6 pm CDT)
+on:
+  pull_request:
+  #schedule:
+  #  - cron: '0 23 * * 0'
+  #push:
+  #  branches:
+  #    - main
+
+jobs:
+  qcow2-mount-inspect:
+    name: Inspect qcow2 image files
+    runs-on: ubuntu-latest
+
+    strategy:
+        matrix:
+          image_name:
+            - CC-Ubuntu24.04
```

Comment:
Right now just checking this 1 image, but for these we can add the list of any/all the images in the container here for this to iterate over them. May be worth adding at least Ubuntu 22 and Centos9. The biggest time sink is downloading the images here to inspect. So the larger GPU ones will just burn a lot more time (worth it? or not?).